components/eamxx/docs/refs.bib

```diff
@@ -123,3 +123,27 @@ @article{Taylor_et20
 note = {e2019MS001783 10.1029/2019MS001783},
 year = {2020}
 }
+
+@techreport{tiedtke_ecmwf_1979,
+	address = {Shinfield Park, Reading},
+	type = {Technical {Report}},
+	title = {{ECMWF} model parameterisation of sub-grid scale processes},
+	language = {en},
+	institution = {ECMWF},
+	author = {Tiedtke, M. and Geleyn, J.-F. and Hollingsworth, A. and Louis, J.-F.},
+	month = jan,
+	year = {1979},
+	note = {10},
+	pages = {146},
+}
+
+@article{raisanen2004stochastic,
```

Comment:
I was trying to add references to the .bib in alphabetical order in order to make them easier to find, but in my .bib PR I asked the question "how long is this going to last?". Apparently 1 PR ;-). I'd prefer you put raisanen in alphabetical order, but am resigned to losing alphabetization.